doc/snippets/Microsoft.Data.SqlClient/SqlColumnEncryptionCngProvider.xml

```diff
@@ -1,67 +1,90 @@
-                The <xref:Microsoft.Data.SqlClient.SqlColumnEncryptionCngProvider.SignColumnMasterKeyMetadata%2A> method must be implemented by the corresponding key store providers. 
-                <xref:Microsoft.Data.SqlClient.SqlColumnEncryptionCngProvider.SignColumnMasterKeyMetadata%2A> should use an asymmetric key identified by a key path and sign the masterkey metadata consisting 
-                of `masterKeyPath`, `allowEnclaveComputations`, and providerName.  
-  
-                ]]></format>
-            </remarks>
-        </SignColumnMasterKeyMetadata>
-        <VerifyColumnMasterKeyMetadata>
-            <param name="masterKeyPath">The complete path of an asymmetric key. The path format is specific to a key store provider.</param>
-            <param name="allowEnclaveComputations">A Boolean that indicates if this key can be sent to the trusted enclave.</param>
-            <param name="signature">The master key metadata signature.</param>
-            <summary>This function must be implemented by the corresponding Key Store providers. This function should use an asymmetric key identified by a key path and verify the masterkey metadata consisting of (masterKeyPath, allowEnclaveComputations, providerName).</summary>
-            <returns>A Boolean that indicates if the master key metadata can be verified based on the provided signature.</returns>
-            <remarks>To be added.</remarks>
-        </VerifyColumnMasterKeyMetadata>
-    </members>
+  <members name="SqlColumnEncryptionCngProvider">
+    <SqlColumnEncryptionCngProvider>
+      <summary>
+        The CMK Store provider implementation for using the Microsoft Cryptography API: Next Generation (CNG) with <see href="https://docs.microsoft.com/sql/relational-databases/security/encryption/always-encrypted-database-engine">Always Encrypted</see>.
+      </summary>
+      <remarks>
+        Enables storing Always Encrypted column master key keys in a store, such as a hardware security module (HSM), that supports the Microsoft Cryptography API: Next Generation (CNG).
+      </remarks>
+    </SqlColumnEncryptionCngProvider>
+    <ctor>
+      <summary>
+        Initializes a new instance of the <see cref="T:Microsoft.Data.SqlClient.SqlColumnEncryptionCngProvider" /> class.
+      </summary>
+    </ctor>
+    <DecryptColumnEncryptionKey>
+      <param name="masterKeyPath">
+        The master key path.
+      </param>
+      <param name="encryptionAlgorithm">
+        The encryption algorithm.
+      </param>
+      <param name="encryptedColumnEncryptionKey">
+        The encrypted column encryption key.
+      </param>
+      <summary>
+        Decrypts the given encrypted value using an asymmetric key specified by the key path and the specified algorithm. The key path will be in the format of <c>[ProviderName]/KeyIdentifier</c> and should be an asymmetric key stored in the specified CNG key store provider. The valid algorithm used to encrypt/decrypt the CEK is 'RSA_OAEP'.
```

Comment:
What is the difference between using `<c>string</c>` or "double quoted string" or 'single quoted string' ? I see some inconsistent usages over RSA_OAEP and strings with slashes '/'- please fix them to use consistent way of quoting.